### PR TITLE
fix: [CDS-88353]: Added learn more link in new nav banner and some other fixes

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
### Summary

Changes in this PR - 

1. Added 'learn more' link on the new nav banner
2. Dismiss banner when User navigates to the new nav
3. Fixed redirection after creating project in Ci and CD

#### Screenshots

Screenshot 2024-01-10 at 6 16 24 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Comment by avinashmadhwani02** [comment]
trigger messagecheck

**Comment by avinashmadhwani02** [comment]
trigger messagecheck

**Comment by avinashmadhwani02** [comment]
trigger messagecheck

**Review by Sagar-Rajput7 — DISMISSED** [review]

**Comment by MondalSayantan** [comment]
Nit: Is double space after the exclamation mark `here!  Gain` intentional? Looks a bit odd to me

File: src/modules/10-common/layouts/DefaultLayout.tsx [hunk 1]
```diff
@@ -126,6 +126,11 @@ export function DefaultLayout(props: React.PropsWithChildren<DefaultLayoutProps>
                 <Text margin={{ right: 'small' }} font={{ variation: FontVariation.SMALL }}>
                   {getString('common.newNavCallout.subtitle')}
                 </Text>
+                <a href="https://youtu.be/KglcP3a9ZaA" rel="noreferrer" target="_blank">
+                  <Text font={{ variation: FontVariation.BODY }} color={Color.PRIMARY_7} margin={{ right: 'small' }}>
```
Comment (abhinavrastogi-harness) [inline src/modules/10-common/layouts/DefaultLayout.tsx:130]: we should add an icon to indicate that this link opens in a new tab

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by litvinovstas — DISMISSED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Review by dhruvUpadhyay07 — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**